packages/react-composites/src/composites/CallComposite/styles/HoldPane.styles.ts

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IButtonStyles, IStackStyles, ITextStyles } from '@fluentui/react';
+import { _pxToRem } from '@internal/acs-ui-common';
+
+/**
+ * styles for hold pane resume button
+ *
+ * @private
+ */
+export const resumeButtonStyles = (): IButtonStyles => {
```

Comment:
nit: don't use functions for these as the result would need to be memoized, instead just use a static object